modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -2,15 +2,15 @@
 // It is subject to the license terms in the LICENSE file found in the top-level directory
 // of this distribution and at http://opencv.org/license.html.
 //
-// Copyright (C) 2018-2023 Intel Corporation
+// Copyright (C) 2018-2024 Intel Corporation
 
 #include "precomp.hpp"
 
 // needs to be included regardless if IE is present or not
 // (cv::gapi::ie::backend() is still there and is defined always)
 #include "backends/ie/giebackend.hpp"
 
-#ifdef HAVE_INF_ENGINE
+#if defined HAVE_INF_ENGINE && INF_ENGINE_RELEASE < 2024000000
```

Comment:
Does it make since to introduce `HAVE_OPENVINO` for versions GE than `2022.1` and set `HAVE_INF_ENGINE` to false for versions GE `2024.0` ?